home.html

```diff
@@ -1,76 +1,342 @@
+            </ul>
+
+            <p>
+              A daily selection of privately personalized reads, no accounts
+              or sign up required.
+            </p>
+            <div class="mt-4"></div>
+            <div>
+              <a href="#" class="btn btn-resume">See the project</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <div class="mt-4"></div>
+      <section class="innerworkcontainer">
+        <div class="work project-1">
+          <div class="description">
+            <h1>Multi-post stories</h1>
+            <ul class="company">
+              <li class="item item1">
+                <h3>Facebook</h3>
+              </li>
+              <li class="item item2">
+                <h3>Full stack dev</h3>
+              </li>
+              <li class="item item2">
+                <h3>2015</h3>
+              </li>
+            </ul>
+
+            <ul class="tools">
+              <li class="item item1">HTML</li>
+              <li class="item item2">Ruby on rails</li>
+              <li class="item item2">CSS</li>
+              <li class="item item2">Javascript</li>
+            </ul>
+
+            <p>
+              Experimental content creation feature that allows users to add
+              to an existing story over the course of a day without spamming
+              their friends.
+            </p>
+            <div class="mt-4"></div>
+            <div>
+              <a href="#" class="btn btn-resume">See the project</a>
+            </div>
+          </div>
+          <div class="project">
+            <img src="assets/project2.png" alt="" />
+          </div>
+        </div>
+      </section>
+
+      <div class="mt-4"></div>
+      <section class="innerworkcontainer">
+        <div class="work project-1">
+          <div class="project">
+            <img src="assets/project3.png" alt="" />
+          </div>
+          <div class="description">
+            <h1>Facebook 360</h1>
+            <ul class="company">
+              <li class="item item1">
+                <h3>Facebook</h3>
+              </li>
+              <li class="item item2">
+                <h3>Full stack dev</h3>
+              </li>
+              <li class="item item2">
+                <h3>2015</h3>
+              </li>
+            </ul>
+
+            <ul class="tools">
+              <li class="item item1">HTML</li>
+              <li class="item item2">CSS</li>
+              <li class="item item2">Javascript</li>
+            </ul>
+
+            <p>
+              Exploring the future of media in Facebook's first Virtual
+              Reality app; a place to discover and enjoy 360 photos and videos
+              on Gear VR.
+            </p>
+            <div class="mt-4"></div>
+            <div>
+              <a href="#" class="btn btn-resume">See the project</a>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <div class="mt-4"></div>
```

Comment:
- [ ] Please remove the empty `div's` I do see a lot of them in the HTML.

Reply:
Thanks, but as you can see, I am using it to set margin-top, if I remove it the design will be ugly.